### PR TITLE
[TIMOB-17998] Android: Deprecation of useSpinner and multipicker

### PR DESCRIPTION
**Pull request by ashcoding**
Jira: https://jira.appcelerator.org/browse/TIMOB-17998

### CODE REVIEW

**Comment by hansemannn** [comment]
Hey Ash, the docs version is 5.2.1, we will need a backport then (to be merged after 5.2.0).

**Comment by ashcoding** [comment]
@hansemannn Yup. Will be waiting after 5.2.0 is tagged off first before doing that backport.

File: android/modules/ui/src/java/ti/modules/titanium/ui/PickerProxy.java [hunk 1]
```diff
@@ -148,11 +149,13 @@ private TiUIView createDateSpinner(Activity activity)
 	@Kroll.getProperty @Kroll.method
 	public boolean getUseSpinner()
 	{
+	    Log.w(TAG, "The useSpinner property is deprecated. Please refer to the documentation for more information");
```
Comment (hieupham007) [inline android/modules/ui/src/java/ti/modules/titanium/ui/PickerProxy.java]: indentation is off?

File: android/modules/ui/src/java/ti/modules/titanium/ui/PickerProxy.java [hunk 2]
```diff
@@ -148,11 +149,13 @@ private TiUIView createDateSpinner(Activity activity)
...
 		return useSpinner;
 	}
 	@Kroll.setProperty @Kroll.method
 	public void setUseSpinner(boolean value)
 	{
+	    Log.w(TAG, "The useSpinner property is deprecated. Please refer to the documentation for more information");
```
Comment (hieupham007) [inline android/modules/ui/src/java/ti/modules/titanium/ui/PickerProxy.java]: same here

**Comment by ashcoding** [comment]
@hieupham007 Indentation addressed.

**Comment by hieupham007** [comment]
Looks good. Merging

**Pull request merged**